app/scripts/requirements

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env php
+<?php
+
+// Script should not be called from the Web
+if (php_sapi_name() != 'cli') {
+    die('Nope');
+}
+
+function colorizeOutput($text, $color)
+    {
+        switch ($color) {
+            case 'green':
+                // White on green
+                $output = "\033[1;37m\033[42m{$text}";
+                break;
+            case 'red':
+                // White on red
+                $output = "\033[1;37m\033[41m{$text}";
+                break;
+            default:
+                $output = $text;
+                break;
+        }
+        $output .= "\033[0m";
+        return $output;
+    }
+
+/**
+ * Checks a configuration.
+ */
+function check($boolean, $message, $help = '', $fatal = false, $fix = '')
+{
+    $text = sprintf("* $message%s\n", $boolean ? '' : ': FAILED');
+
+    if (! $boolean) {
+        if(! empty($help)) { 
+            $text .=  "  $help\n";
+        }
+        if(! empty($fix)) {
+                $text .=  "  $fix\n";
+        }
+    }
+    if($boolean) {
+        echo colorizeOutput($text, 'green');
+    } else {
+        echo colorizeOutput($text, 'red');
+    }
+    if (! $boolean && $fatal)
+    {
+        die("You must fix this problem before resuming the check.\n");
```

Comment:
Why do we need to die for a fatal error?

I would also simplify the amount of ifs here
```PHP
if (! $boolean) {
    if(! empty($help)) {
        $text .=  "  {$help}\n";
    }
    if(! empty($fix)) {
        $text .=  "  {$fix}\n";
    }
    echo colorizeOutput($text, 'red');
} else {
    echo colorizeOutput($text, 'green');
}
```